tests/sif/translation/test_while.py

```diff
@@ -0,0 +1,44 @@
+from nagini_contracts.contracts import *
```

Comment:
So a), it's great that you already have tests.
b), generally it's a good idea to always have a version of each test that is supposed to fail, just to check that we cannot prove things that are false. You'd need to use the #:: ExpectedOutput annotations that are also in other test cases then. A nice side effect is that you test the error reporting in the process.